table/namespace_classifier.go

```diff
@@ -123,6 +123,7 @@ func (c *tableNamespaceClassifier) GetAllNamespaces() []string {
 	for name := range c.nsInfo.namespaces {
 		nsList = append(nsList, name)
 	}
+	nsList = append(nsList, namespace.DefaultNamespace)
```

Comment:
need to check duplicated default here?

Reply:
Maybe forbid creating namespace using `global` instead.